settings/src/main/kotlin/voice/settings/views/AutoSleepTimerSetting.kt

```diff
@@ -0,0 +1,48 @@
+import androidx.compose.material3.LocalContentColor
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.material3.TimePickerState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import voice.settings.views.TimePickerDialog
+import java.time.LocalTime
+
+@Composable
+internal fun AutoSleepTimerSetting(
+  autoSleepTimer: Boolean,
+  time: String,
+  label: String,
+  setAutoSleepTime: (Int, Int) -> Unit,
+) {
+  val shouldShowTimePicker = remember { mutableStateOf(false) }
+  CompositionLocalProvider(
+    LocalContentColor provides
+      if (autoSleepTimer) MaterialTheme.colorScheme.onSurface
+      else MaterialTheme.colorScheme.onSurface.copy(alpha = 0.38f),
```

Comment:
You can follow the `AppVersion` composable to achieve a consistent disabled look